src/components/ArticleList.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import SyncLoader from "react-spinners/SyncLoader";
+import { getArticles } from "../utils/api";
+import ArticleListItem from "./ArticleListItem";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  useEffect(() => {
+    setIsLoading(true);
+    getArticles().then((articles) => {
+      setArticles(articles);
+      setIsLoading(false);
+    });
+  }, []);
+  return (
+    <div className="articles__container">
```

Comment:
This component should be a `main` or `section`. It's a distinct part of the site with some actual meaning rather than just a div 👍